### DISCUSSION TITLE
Feature Request: Support configuring width/height for quick terminal

### DISCUSSION THREAD

**Discussion by Integralist** [ghostty-org/ghostty#3697] [Feature Requests, Ideas] [answered]
I'm not sure if this is possible with the macOS quick terminal implementation, but I'd really like to be able to be able to configure the width and height of the quick terminal window (if you look at the screenshot below, it's just too small for my large screen, also my prompt gets broken on smaller screens 😅).

Something like:

```
quick-terminal-position = center
quick-terminal-width = <UNIT?>
quick-terminal-height = <UNIT?>
```

<img width="1920" alt="Screenshot 2024-12-28 at 11 06 02" src="https://github.com/user-attachments/assets/3d98eea6-a9f4-43a7-8ae0-cdd9ec0850bf" />

> [!NOTE]
> **UPDATE:** I now see there is a duplicate [here](https://github.com/ghostty-org/ghostty/discussions/3524) which asks for separate configuration for the quick terminal. So I can close this.

**Comment by 00-kat** [comment, marked as answer]
Duplicate of #2384.

**Reply by Integralist** [reply]
Yeah I just noticed that myself (see my UPDATE note) 👍🏻

**Reply by 00-kat** [reply]
Yep I did, but the post you linked wasn't the one it was a duplicate of.
